test/spec/fingerprint.js

```diff
@@ -145,5 +146,51 @@ define(function(require) {
         done();
       });
     });
+
+    util.itMakesCorrectRequestResponse({
+      title: 'attaches fingerprint to signIn requests if addFingerprint is true',
+      setup: {
+        uri: 'http://example.okta.com',
+        calls: [
+          {
+            request: {
+              method: 'post',
+              uri: '/api/v1/authn',
+              data: { username: 'not', password: 'real' },
+              headers: {
+                'Accept': 'application/json',
+                'Content-Type': 'application/json',
+                'X-Okta-User-Agent-Extended': 'okta-auth-js-' + packageJson.version,
+                'X-Device-Fingerprint': 'ABCD'
+              }
+            },
+            response: 'success'
+          }
+        ]
+      },
+      execute: function (test) {
+        return setup({ authClient: test.oa }).signIn({
+          username: 'not',
+          password: 'real',
+          sendFingerprint: true
```

Comment:
not sure if we have one test already, but if not maybe we should add one with ```sendFingerprint: false``` or ```sendFingerptint: undefined```

Reply:
We have test for signIn that don't pass `sendFingerprint`, but not one that sends `false`